body.tex

```diff
@@ -490,35 +597,10 @@ \subsubsection{Completing the Work}
 \item (If applicable, i.e. it is not a bucket, as defined in \S\ref{sec:bucket}) its concrete deliverable has been achieved.
 \end{enumerate}
 
+``Bucket'' epics should be closed at the end of the time box (i.e. end of half fiscal year and end of fiscal year).
 Note that it is not permitted to close an \gls{epic} without defining at least one \gls{story} within it.
 Empty \glspl{epic} can never be completed.
 
-\subsection{Handling Bugs \& Emergent Work}
```

Comment:
if this goes out perhaps we should refer to developer.io ??  https://developer.lsst.io/work/flow.html

Reply:
This section can be included if you want, but with a note that it is for DP and SF, not ROO or REO (at this time).

Reply:
Refer to developer.io -- and we should agree how this is done for all..